pkg/config/parse.go

```diff
@@ -0,0 +1,99 @@
+package config
+
+import (
+	"io/ioutil"
+	"runtime"
+
+	"github.com/BurntSushi/toml"
+	"github.com/kelseyhightower/envconfig"
+	validator "gopkg.in/go-playground/validator.v9"
+)
+
+var validate *validator.Validate
+
+func getValidator() *validator.Validate {
```

Comment:
this is a little bit magical. If the *global* variable is not nil then return it, otherwise return a new one. Not sure why we have to set up things this way. We can just initialize `validate` globally on line 12 like this: `var validate = validator.New()` and that's it. Or, we can remove this function as well as the global variable and just initialize a new validator when we wanna use it in other funcs.

Reply:
good point :)